src/components/store-connector/index.js

```diff
@@ -0,0 +1,48 @@
+import React, {PropTypes} from 'react';
+import connectToStore from 'focus-components/behaviours/store/connect';
+
+// PropTypes
+const propTypes = {
+    children: PropTypes.element,
+    storeProperties:
+    PropTypes.arrayOf(
+        PropTypes.shape({
+            store: PropTypes.any.isRequired,
+            properties: PropTypes.arrayOf(PropTypes.string)
+        })
+    ),
+    storeGetFunction: PropTypes.func.isRequired,
+    component: PropTypes.func.isRequired
+};
+// defaultProps
+const defaultProps = {
+};
+//displayName
+const displayName = 'StoreConnector';
+
+const StoreConnector = ({storeProperties, storeGetFunction, component, ...others}) => {
+    const ConnectedComp = connectToStore(storeProperties, storeGetFunction)(component);
```

Comment:
@Hartorn What you are doing is an other pattern, you can have :

- high order component which is `connectToStore`
- child function component which is the `component` props

These pattern are both correct but i am not comfortable with enforcing two way of doing the same thing.
Even if they are even regarding their behaviour.